pkg/annotations/annotations.go

```diff
@@ -192,6 +192,9 @@ const (
 	// VSMBNoDirectMap specifies that no direct mapping should be used for any VSMBs added to the UVM
 	VSMBNoDirectMap = "io.microsoft.virtualmachine.wcow.virtualSMB.nodirectmap"
 
+	// DisableWriteableFileShares disables adding any writable fileshares to the UVM
+	DisableWriteableFileShares = "io.microsoft.virtualmachine.fileshares.disablewritable"
```

Comment:
Writeable and writable are interchanged here for the name of the variable and the annotation string. Should be writable for both